vulnerabilities/importer_registry.json

```diff
@@ -0,0 +1,182 @@
+[


```

Comment:
Looks like the line endings here need fixing.

Reply:
I don't understand what you mean.
Can you elaborate ? Do you mean ^M , that is not me ;) it's a stupid way of rendering json github uses, try raw mode.

Reply:
> Can you elaborate ? Do you mean ^M , that is not me ;) it's a stupid way of rendering json github uses, try raw mode.

Interesting. Never seen that before. Looks like DOS line endings. nvm then :)

Reply:
The line endings are DOS indeed. We should use LF *nix instead:
```
$ wget https://raw.githubusercontent.com/nexB/vulnerablecode/7bfa1c57d3cb4846067ff59ca2d6cde667fee29d/vulnerabilities/importer_registry.json
$ file importer_registry.json 
importer_registry.json: ASCII text, with CRLF, CR line terminators
```